packages/contracts-bedrock/src/L2/OptimismSuperchainERC20.sol

```diff
@@ -31,10 +22,13 @@ error ZeroAddress();
 ///         token, turning it fungible and interoperable across the superchain. Likewise, it also enables the inverse
 ///         conversion path.
 ///         Moreover, it builds on top of the L2ToL2CrossDomainMessenger for both replay protection and domain binding.
-contract OptimismSuperchainERC20 is IOptimismSuperchainERC20Extension, ERC20, ISemver, Initializable, ERC165 {
-    /// @notice Address of the L2ToL2CrossDomainMessenger Predeploy.
-    address internal constant MESSENGER = Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER;
-
+contract OptimismSuperchainERC20 is
+    IOptimismSuperchainERC20Extension,
```

Comment:
Does this need `IOptimismSuperchainERC20Extension` if its `SuperchainERC20`?

Reply:
We made a distinction between the interfaces, this one also have the `mint` and `burn` functionality used by the `L2StandardBridge`. The `ISuperchainERC20` only have the send/relay one.